Assets/MRTK/Tests/PlayModeTests/Experimental/BoundsControlTests.cs

```diff
@@ -268,6 +269,41 @@ public IEnumerator ScaleViaNearInteraction()
             yield return null;
         }
 
+        /// <summary>
+        /// Uses near interaction to scale the bounds control by directly grabbing corner - precise scaling 
+        /// </summary>
+        [UnityTest]
+        public IEnumerator ScaleNonUniform()
+        {
+            BoundsControl boundsControl = InstantiateSceneAndDefaultBoundsControl();
+            boundsControl.ScaleHandlesConfig.ScaleBehavior = HandleScaleMode.NonUniform;
+            yield return VerifyInitialBoundsCorrect(boundsControl);
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+
+            // front right corner is corner 3
+            var frontRightCornerPos = boundsControl.gameObject.transform.Find("rigRoot/corner_3").position;
+
+            Vector3 initialHandPosition = new Vector3(0, 0, 0.5f);
+            var delta = new Vector3(0.1f, 0.1f, 0f);
+            TestHand hand = new TestHand(Handedness.Left);
+            yield return hand.Show(initialHandPosition);
+            yield return hand.MoveTo(frontRightCornerPos);
+            yield return hand.SetGesture(ArticulatedHandPose.GestureId.Pinch);
+            yield return hand.MoveTo(frontRightCornerPos + delta);
+            yield return null;
+
+            var endBounds = boundsControl.GetComponent<BoxCollider>().bounds;
+            Vector3 expectedCenter = new Vector3(0.05f, 0.05f, 1.5f);
+            Vector3 expectedSize = Vector3.one * .6f;
+            expectedSize.z = 0.5f;
+            TestUtilities.AssertAboutEqual(endBounds.center, expectedCenter, "endBounds incorrect center");
+            TestUtilities.AssertAboutEqual(endBounds.size, expectedSize, "endBounds incorrect size");
+            
+            GameObject.Destroy(boundsControl.gameObject);
```

Comment:
Out of curiosity, the scene game objects are being destroyed on the test TearDown inside PlayModeTestUtilities.TearDown(). Is there a special need for calling this here?

Reply:
no we don't need it. i'll remove it